token/token.go

```diff
@@ -454,6 +454,13 @@ func IsSpecificServiceAccount(ctx context.Context, names []string) bool {
 	return false
 }
 
+// IsServiceAccount checks if the request is done by a
+// Service account based on the JWT Token provided in context
+func IsServiceAccount(ctx context.Context) bool {
```

Comment:
Add a test for this function.

Reply:
I've added a couple of tests for this method to the token test suite.